common/templates/context.go

```diff
@@ -957,6 +957,16 @@ func (s Slice) Append(item interface{}) (interface{}, error) {
 	}
 }
 
+func (s Slice) Del(index int) (interface{}, error) {
+	if index >= len(s) {
```

Comment:
`index < 0` needs to be checked as well; the fact that Set leaves this out is a mistake (and results in a runtime panic, though one that is recovered.)